<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralConfigurationDialog</class>
 <widget class="QWidget" name="GeneralConfigurationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GeneralConfigurationDialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>240</y>
     <width>195</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="myButtonOk">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="myButtonCancel">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>531</width>
     <height>91</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="title">
    <string>Filter</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>511</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0">
     <item>
      <widget class="QCheckBox" name="myCheckboxUseFilter">
       <property name="text">
        <string>Use reconstruction filter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="myComboFilterType">
       <item>
        <property name="text">
         <string>Box</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Gaussian</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MitchellNetravali</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="myButtonConfigFilter">
       <property name="text">
        <string>Configure</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>531</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Computation</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>227</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Nb. threads</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="mySpinNbThreads">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Oversampling</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="mySpinOversampling">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Block size</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="mySpinBlockSize">
         <property name="minimum">
          <number>2</number>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
         <property name="value">
          <number>32</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
